Description: Install pdf on non-MIPS architectures.
Forwarded: not-needed
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2016-11-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/docsrc/Makefile.am
+++ b/doc/docsrc/Makefile.am
@@ -1,3 +1,9 @@
+if DISABLE_DOC_BUILD
+
+regen:
+	echo Skipping
+
+else
 
 XELATEX_FLAGS = -halt-on-error -file-line-error
 XELATEX = xelatex
@@ -9,4 +15,12 @@
 
 EXTRA_DIST = $(srcdir)/*.tex
 
+install-data-local:
+	mkdir -p $(DESTDIR)$(datadir)/doc/asn1c-doc
+	cp -a asn1c-usage.pdf $(DESTDIR)$(datadir)/doc/asn1c-doc
+
+uninstall-local:
+	rm -f $(DESTDIR)$(docsdir)/asn1c-usage.pdf
+
 CLEANFILES = *.*~ *.aux *.dvi *.log *.out *.toc
+endif
--- a/configure.ac
+++ b/configure.ac
@@ -26,6 +26,9 @@
 	;;
 esac
 
+# MIPS build failed due to #836454
+AM_CONDITIONAL([DISABLE_DOC_BUILD], [test x$host_cpu = xmips])
+
 AC_ARG_ENABLE(Werror,
   [  --enable-Werror         abort compilation after any C compiler warning],
 	enable_werror=$enableval, enable_werror=no)
